clients/src/main/java/org/apache/kafka/common/requests/AbstractResponse.java

```diff
@@ -268,6 +268,8 @@ public ApiKeys apiKey() {
 
     public abstract int throttleTimeMs();
 
+    public abstract void setThrottleTimeMs(int throttleTimeMs);
```

Comment:
How about `maybeSetThrottleTimeMs` since not all response schemas support it.

Reply:
Makes sense. Will change.